docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,141 @@
+# napari 0.3.0
+
+We're happy to announce the release of napari 0.3.0!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+
+## New Features
+- Hook up reader plugins (#937)
+- Support for magicgui (#981)
+- Writer plugins (#1104)
+
+
+## Improvements
+- Generalize keybindings (#791)
+- Points view data refactor (#951)
+- Add magic name guessing (#1008)
```

Comment:
Oh man, I forgot this merged! Again, so damn cool, @kne42! It might be worth highlighting!

Reply:
I think now that it should be the default, ie we should remove the check for `os.environ['MAGICNAME']` here:

https://github.com/napari/napari/blob/6933f5bcc12608d66458987d69db2f7c535b21de/napari/layers/base/base.py#L136

Thoughts @sofroniewn @tlambert03 @kne42?

Reply:
ah do we need to do this before we make the release given the paragraph on this functionality in the new release. @kne42 can you turn this on by default?

Reply:
@sofroniewn @kne42 see #1186

Reply:
this is now done